### PR TITLE
install to /usr/libexec/docker and fixes

### PR DESCRIPTION
**Pull request by runcom**
Fix https://github.com/shishir-a412ed/docker-lvm-plugin/issues/23

Signed-off-by: Antonio Murdaca runcom@redhat.com

### CODE REVIEW

**Comment by runcom** [comment]
The second commit cleanup and fix some stuff left /cc @rhatdan

**Comment by runcom** [comment]
I'm facing a lockup of the docker create command when no vggroup is configured in the conf... @shishir-a412ed any hint?

File: systemd/docker-lvm-plugin.service [hunk 1]
```diff
@@ -5,8 +5,8 @@ Before=docker.service
 
 [Service]
 Type=simple
-EnvironmentFile=-/etc/sysconfig/docker-lvm-volumegroup
-ExecStart=/usr/bin/docker-lvm-plugin
+EnvironmentFile=-/etc/sysconfig/docker-lvm-plugin
```
Comment (shishir-a412ed) [inline systemd/docker-lvm-plugin.service]: This should be `/etc/docker/docker-lvm-plugin`

Comment (runcom) [inline systemd/docker-lvm-plugin.service]: I didn't modify it, I just fixed the suffix (`s/volumegroup/docker-lvm-plugin/`)

Comment (runcom) [inline systemd/docker-lvm-plugin.service]: I like it under sysconfig like the other files such as docker,docker-storage-setup and the like

Comment (shishir-a412ed) [inline systemd/docker-lvm-plugin.service]: I know, this was already incorrect. Can you fix it ?
I think I already had a discussion with @rhatdan on this, and we finalized on `/etc/docker`.

Comment (runcom) [inline systemd/docker-lvm-plugin.service]: ack

Comment (runcom) [inline systemd/docker-lvm-plugin.service]: fixed, could you test everything out?

Comment (shishir-a412ed) [inline systemd/docker-lvm-plugin.service]: Yeah will do.

**Comment by shishir-a412ed** [comment]
LGTM.
Thanks @runcom

**Comment by rhatdan** [comment]
My point was this should not be used in the systemd unit file at all.  docker-lvm-plugin should read in the /etc/docker-lvm-plugin.conf directly and use it.  We should not be passing options using environment variables. 

We do this with docker as a hack, because docker does not support a configuration file.

**Comment by runcom** [comment]
make sense @shishir-a412ed would you take care of entirely removing the Env file from the .service? I can also create an issue

**Comment by shishir-a412ed** [comment]
@rhatdan @runcom 
Actually `docker-lvm-plugin` is reading it directly in the go code. So we don't need to mention it in the systemd unit file.
We can remove it, and I believe the code will still work correctly.

**Pull request merged**